.github/workflows/build.yml

```diff
@@ -6,7 +6,7 @@ env:
   ERT_SHOW_BACKTRACE: 1
 
 jobs:
-  build:
+  build-test-deploy:
```

Comment:
Deploy is a separate job further down

Reply:
Thanks! Well spotted 👍🏻 I also had to update the `needs` argument for `publish`...